.devcontainer/devcontainer.json

```diff
@@ -13,14 +12,14 @@
         "terminal.integrated.profiles.linux": {
           "bash": { "path": "/bin/bash" }
         },
-        "terminal.integrated.defaultProfile.linux": "bash"
+        "terminal.integrated.defaultProfile.linux": "bash",
+        "extensions.ignoreRecommendations": true,
+        "git.autofetch": true
       },
       "extensions": [
         "dbaeumer.vscode-eslint",
         "esbenp.prettier-vscode",
-        "orta.vscode-jest",
```

Comment:
instead we should disable jest autorun

Reply:
Tried that with no avail

Reply:
https://github.com/renovatebot/renovate/blob/932012fa20f2e614b11c05fd896c54e00d41d70b/.vscode/settings.json#L19

doesn't that setting solve the auto run?

Reply:
nop

Reply:
Apparently `jest.autoRun` is an object type, not a string, and `off` is short for `{"watch": false}`[^1]. Can you try `"jest.autoRun": {}`?

[^1]: https://github.com/jest-community/vscode-jest#autorun

Reply:
Tried. Same result.